### PR TITLE
Optimize CDN GC Debug log

### PR DESCRIPTION
**Pull request by 244372610**
Signed-off-by: sunwp <244372610@qq.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] Code compiles correctly.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by gaius-qi — CHANGES_REQUESTED** [review]
Fix lint

**Review by gaius-qi — DISMISSED** [review]
LGTM

File: cdn/supervisor/cdn/storage/disk/disk_test.go [hunk 1]
```diff
@@ -132,3 +137,27 @@ func (suite *DiskStorageManagerSuite) TestTryFreeSpace() {
 		})
 	}
 }
+
+func (suite *DiskStorageManagerSuite) TestDeleteTask() {
+	workHome, _ := os.MkdirTemp("/tmp", "cdn-storageManager-")
+	fmt.Printf("workHome: %s", workHome)
```
Comment (KubeStacker) [inline cdn/supervisor/cdn/storage/disk/disk_test.go:143]: remove this？

Comment (KubeStacker) [reply cdn/supervisor/cdn/storage/disk/disk_test.go:143]: remove fmt.Printf?

**Review by KubeStacker — APPROVED** [review]

**Review by gaius-qi — APPROVED** [review]
LGTM

**Pull request merged**